FilmPortal/FilmPotal.WEB/App/reducers/filmReducer.jsx

```diff
@@ -0,0 +1,48 @@
+import { GET_FILM_SUCCESS, DELETE_COMMENT_SUCCESS, DELETE_COMMENT_ERROR, GET_FILM_ERROR, ADD_COMMENT_SUCCESS, ADD_COMMENT_ERROR, CHANGE_COMMENT_TEXT, CHANGE_RATING_ERROR, CHANGE_RATING_SUCCESS, GET_MARK_SUCCESS, GET_MARK_ERROR } from '../actions/filmActions.jsx'
+
+const initialState = {
+    film: { comments: [], images:[] },
+    comment: '',
+    rating: '',
+    error: ''
+}
+
+export default function film(state = initialState, action) {
+    switch (action.type) {
+        case GET_FILM_SUCCESS:
```

Comment:
It's good practice to have 3 events: `Action_Begin`, `Action_Success`, `Action_Failure`